deployment/entity-service/README.md

```diff
@@ -0,0 +1,29 @@
+Few helm tips:
+
+To check if the provided values are sufficient, remove/delete the `Charts` folder and run:
```

Comment:
I do not have the impression it can point to a released chart. The help:
```
This command takes a path to a chart and runs a series of tests to verify that
the chart is well-formed.
```
And when simply running `helm lint` from a random folder:
```
$ helm lint
==> Skipping .
No chart found for linting (missing Chart.yaml)

Error: 0 chart(s) linted, 1 chart(s) failed
```
which shows that it is searching for a `Chart.yaml`.